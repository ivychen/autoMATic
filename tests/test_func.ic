int fun(int a, int b) {
    return a + b;
}

int main() {
    print(fun(1, 2)); // 3
    print(fun(-600, 10)); // -590
    print(fun(12345, 67890)); // 80235
    return 0;
}
