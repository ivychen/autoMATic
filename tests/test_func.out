3
-590
80235
